build/stf-run-ci/tasks/main.yml

```diff
@@ -76,7 +95,33 @@
     tags:
       - deploy
 
-- block:
+- when: __deploy_from_index_enabled | bool
+  tags:
+    - create_bundles
+  block:
+    - name: Create base build list
```

Comment:
I started to move this and then got wondering if there is a use case here where you'd want local builds + bundles without the index image?

Effectively the index-based deployment provides this (although with a bit more overhead of creating the index image and single CatalogSource vs bundle deployment which creates separate CatalogSources for each of the bundles).

Maybe I'm missing it, but I'm wondering if it's actually useful to add `__deploy_from_bundles_enabled` + `__local_build_enabled` at this point?

Reply:
`__deploy_from_bundles_enabled` is required because downstream we have bundles with no associated index image, and we need to install them for testing. I have imagined wanting to install exactly the bundles being tested, but with a local build of one of the components, in order to test a fix.